SCons/Util.py

```diff
@@ -319,14 +343,11 @@ def print_tree(root, child_func, prune=0, showtags=0, margin=[0], visited=None,
     def MMM(m):
         if singleLineDraw:
             return ["  ", BOX_VERT + " "][m]
-        else:
-            return ["  ", "| "][m]
 
-    margins = list(map(MMM, margin[:-1]))
+        return ["  ", "| "][m]
```

Comment:
Checker suggests removing the else?

Reply:
yes, since you return unconditionally in the "if", what's left is necessarily the "else" part, so the else itself is not needed. (hey, I didn't write the checker :) )